tests/integration/features/backup_delete.feature

```diff
@@ -110,6 +110,45 @@ Feature: Backup & Clean & Restore
       | 5   | created  | 4          | 4            | shared+links  |
       | 6   | created  | 4          | 0            | shared        |
 
+  Scenario: Create "unusual_char" backup
+    When we execute command on clickhouse01
+    """
+    echo -e 'CREATE DATABASE unusual_char;' | clickhouse-client
+    """
+    And we execute command on clickhouse01
+    """
+    echo -e 'CREATE TABLE unusual_char."tab\x07l./-_e" (n UInt64) ENGINE = MergeTree() ORDER BY n;' | clickhouse-client
+    """
+    And we execute command on clickhouse01
+    """
+    echo -e 'INSERT INTO unusual_char."tab\x07l./-_e" VALUES (1234);' | clickhouse-client
```

Comment:
Can we use `execute query` or `clickhouse-client --query` or it is impossible because of unusual chars?